### DISCUSSION TITLE
Lifting providers out of the repo

### DISCUSSION THREAD

**Discussion by micahkendall** [butaneprotocol/translucent-compat#6] [General]
If we don't want community to be burdened with maintaining compatibility with any of the providers, we can potentially lift any unmaintained providers out as seperate repos which are pinned to an older version of translucent. This would be a better style as well if we want to mix and match many providers, and make it easy for others to bring new external providers to translucent without fumbling through bureaucracy of any review process.

**Comment by joacohoyos** [comment]
I like the idea , the way it is right now given that the provider is just an interface we already support using `external` providers without them necesarilly being in the repo

**Comment by rvcas** [comment]
I suggest going down the monorepo route and maintaining a few default providers but we could also encourage that paid services (maestro/blockfrost) maintain their own providers (in their own repos and npm packages). If the tools are open source such as kupo and Ogmios I think it's nice to maintain those in repo/monorepo.
